### DISCUSSION TITLE
Endosomal clearance of ligand-FcRn complex in the large molecule model

### DISCUSSION THREAD

**Discussion by moPellowe** [Open-Systems-Pharmacology/Forum#1431] [Questions & Problems] [answered]
I have a question about the passive transport EndosomalSpecificClearance_FirstOrder within the large molecule model. From what I can see, it looks like the drug, FcRn, LigandEndo, and LigandEndo-FcRn_Complex are all subject to this passive transport representing endosomal clearance. Shouldn't the LigandEndo_Complex also be excluded from this passive transport?

![image](https://user-images.githubusercontent.com/57540473/211020089-2114ff68-eb50-4592-b6e5-3a7801aeb929.png)

**Comment by Christoph27** [comment, marked as answer]
You are right, that the "EndosomalSpecificClearance_FirstOrder" transport should not transport "LigandEndo_Complex".
And the transport is indeed not transporting "LigandEndo_Complex" since "LigandEndo_Complex" is set to be not present in the compartment "EndosomalClearance" in the building block "Molecule Start Values".

**Reply by moPellowe** [reply]
Oh, okay. That makes sense. Thank you for clarifying that.
